### ISSUE TITLE
Unable to handle a function which returns a tuple

### ISSUE THREAD

**Issue by barakman** [sc-forks/solidity-coverage#188]
It seems that 'solidity-coverage' is unable to handle a function which returns a tuple:

    AssertionError: (intermediate value) is not iterable

The exact same code is handled well by 'truffle':

**Contract:**

```
pragma solidity ^0.4.18;

contract DataSource {
    struct Interval {
        uint256 a;
        uint256 b;
        uint256 c;
        uint256 d;
        uint256 e;
        uint256 f;
    }

    Interval[10] private intervals;

    function getInterval(uint256 index) external constant returns (uint256, uint256, uint256, uint256, uint256, uint256) {
        Interval storage interval = intervals[index];
        return (interval.a, interval.b, interval.c, interval.d, interval.e, interval.f);
    }
}
```

**Test Script:**

```
contract("DataSourceTest", () => {
    describe(`Test:`, async () => {
        let hDataSource;
        before(async () => {
            hDataSource = await artifacts.require("DataSource.sol").new();
        });
        it(`getInterval`, async () => {
            try {
                let [a, b, c, d, e, f] = await hDataSource.getInterval(0);
                let output = [a, b, c, d, e, f].join();
                assert(output == "0,0,0,0,0,0", `output = ${output}`);
            }
            catch (error) {
                assert(false, error.message);
            }
        });
    });
});
```

**Comment by barakman** [comment]
@cgewecke:

NP.

I still have this type of code in my tests, and they run smoothly on both `truffle test` and `solidity-coverage`.
So I am left to assume that it was resolved at some point between the `solidity-coverage` version which I was using at the time (not specified here unfortunately) and the one which I am using now (0.5.4).

I am unable to precisely reproduce the old version at the time of posting this issue (Feb 1) because we've switched to a different repository since (March 28), but for all it matters, my initial `package.json` commit shows `"solidity-coverage": "^0.4.9"`, so it was possibly somewhere around that.

Thanks
